src/test/resources/features/RetrieveReviews.feature

```diff
@@ -14,11 +15,37 @@ Feature: Retrieve review
     When I list all the reviews
     Then The response code is 200
 
-   Scenario: Show a review by id as a user
-     Given I can login with username "user0" and password "password0"
-     When I list the review with id 2
-     Then The response code is 200
-     
+  Scenario: Show a review by id as a user
+    Given I can login with username "user0" and password "password0"
+    When I list the review with id 2
+    Then The response code is 200
```

Comment:
You should at least check that the HttpRequest returns the correct amount of Reviews.